src/blog/2023/11/dashboard-0-7.md

```diff
@@ -0,0 +1,159 @@
+
+In the short term, we'll be keeping @flowforge available on the Node-RED Palette Manager, but it will be removed soon, and the associated NPM Package will be put into a "deprecated" mode.
+
+### NPM Package Migration
+
+Unfortunately, this migration from `@flowforge/` to `@flowfuse/` requires a little bit of manual work, and isn't as easy as just clicking "update" in the Node-RED Editor.
+
+In any case, there is no need to update your flow in this migration, you'll just need to uninstall `@flowforge/node-red-dashboard` and install `@flowfuse/node-red-dashboard` instead.
+
+#### Running Locally
+
+If you're running Node-RED locally, or in your own infrastructure, you'll need to manually uninstall the old package:
+
+```bash
+npm uninstall @flowforge/node-red-dashboard
+```
+
+and re-install the new one:
+
+```bash
+npm install @flowfuse/node-red-dashboard
+```
+
+#### Running in FlowFuse
+
+Navigate to your Instance > Settings > Palette, and then change the `@flowforge/node-red-dashboard` entry to `@flowfuse/node-red-dashboard` (the latest version as of this post is `0.7.2`).
+
+Restart your instance, and the new package will automatically install.
+
+## Dashboard 1.0 to 2.0 Migration Guide
+
+As part of our mission to ensure a smooth transition from Dashboard 1.0 to Dashboard 2.0, we have published a first draft of a [Migration Guide](https://dashboard.flowfuse.com/user/migration.html).
+
+As a starting point, we have comprehesively covered the Dashboard 1.0 widgets and their associated properties. We've then detailed which properties are already supported, which have partial support, and where appropriate, we do not support and _why_ (most of the time, it's just because we haven't got round to it yet!)
+
+![Migration Guide Snippet](./images/migration-guide-snippet.png)
+
+We fully appreciate that the migration path is not yet complete, and we that we are missing some features and properties, but please know that we are working hard to ensure that as many of the features from Dashboard 1.0 are available in Dashboard 2.0. We will be updating this guide as we progress. 
+
+We will also be adding "event" and "dynamic properties" sections to the guide, to detail how you can update and control elements via runtime messages (e.g. dynamically change the label of a button), and how this differs (if at all) from Dashboard 1.0.
+
+Whilst we aren't quite there yet, this guide offers a comprehensive breakdown on our progress in backporting all of the properties from Dashboard 1.0
+
+#### Automated Script
+
+An ambitious plan that we have is to also provide a [Migration Script](https://github.com/FlowFuse/node-red-dashboard/issues/261). Any feedback, ideas or concerns are most welcome as comments on the issue.
```

Comment:
While I get this, I think if you're not going to get around to previous features this should not be mentioned until it comes to fruition?

Reply:
Yeah, it's just been asked for by _so many_ people, I feel it has to be mentioned to save the guaranteed bombardment of questions asking for it